.rubocop.yml

```diff
@@ -310,3 +310,6 @@ Performance/OpenStruct:
 
 Minitest/UnreachableAssertion:
   Enabled: true
+
+Minitest/SkipEnsure:
+  Enabled: true
```

Comment:
That’s just my two cents. It seems to be sorted by ASCII, so it would be better to move it above `Minitest/UnreachableAssertion` cop.

Reply:
Good catch. Fixed.